HalleDMG/DMGYi135IIIC11b.xml

```diff
@@ -0,0 +1,170 @@
+                      </origin>
+                        <provenance>The letter is part of the Ethiopianist ‘Nachlass’  of <persName ref="PRS7812Peterman"/>, which
+                            was handed over to <persName ref="PRS7939Praetori"/> (see    <bibl><ptr target="bm:Praetorius1879Amharisch"/><citedRange unit="page">11</citedRange></bibl>).
+                            </provenance>
+                        <acquisition>It is now kept as part of
+                            the ‘Nachlass Praetorius’ (shelfmark DMG Yi 135) at
+                            <placeName ref="INS0385ULBHalle">Universitäts- und Landesbibliothek Sachsen-Anhalt, Halle (Saale)</placeName> –
+                            <placeName ref="INS0384Bibliothek_der_Deutschen_Morgenlaendischen_Gesellschaft">Bibliothek der Deutschen 
+                                Morgenländischen Gesellschaft</placeName>.</acquisition>
+                    </history>
+                    
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                  <listBibl type="secondary">
+                                      <bibl><ptr target="bm:Praetorius1879Amharisch"/><citedRange unit="page">11</citedRange></bibl>
+                                      <bibl><ptr target="bm:Guhlmann2023Praetorius"/></bibl>
+                                  </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Letter"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="am">Amharic</language>
+                <language ident="ar">Arabic</language>
+                <language ident="de">German</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2024-01-16">Created record after consultation of the manuscript on 21.12.2023</change>
+        </revisionDesc>
+    </teiHeader>
+    <text xml:base="https://betamasaheft.eu/">
+        <body>
+            <div type="edition">
+                <div type="textpart" xml:lang="ar">
```

Comment:
am?

Reply:
other than the first word, the text is in Arabic

Reply:
I have not found anything 100% fitting in TEI but I guess what they suggest is to use a separate language tag with a sub-type for the script (such as `az-Arab` for Azeri in Arabic script as opposed to `az-Lat` for Azeri in Latin script https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-language.html), which would mean adding something like `ar-Eth` for Arabic in fidel, and then in `langUsage `refer to this specific tag 

Otherwise I wonder if we can use `style `as described https://www.tei-c.org/release/doc/tei-p5-doc/en/html/WD.html (here it is mostly used to describe RL LR etc) and have something like
`xml:lang="ar" style="script: Ethiopic"` (it would not be displayed at the moment but at least it would be somewhere)